rss/table_rss_item.go

```diff
@@ -49,7 +50,12 @@ func listItem(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (
 	feed, err := fp.ParseURLWithContext(fl, ctx)
 	if err != nil {
 		plugin.Logger(ctx).Error("listItem", "Error", err)
-		return nil, err
+		if strings.Contains(err.Error(), "x509: certificate") {
+			// Return an empty array of items
+			return []gofeed.Item{}, nil
+		} else {
+			return nil, err
+		}
```

Comment:
@judell Do you have the full error we get for this case?

Reply:
I will see if I can find which server threw that error.

Reply:
https://mastodon.think-privacy.com/@aurelie.rss: x509: certificate has expired or is not yet valid: current time 2023-05-17T10:21:02-07:00 is after 2023-04-12T06:37:47Z